xbmc/linux/sse4/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
-set(SOURCES CopyFrame.cpp)
+if(HAVE_SSE4_1)
```

Comment:
@mapfau: Might be interesting for you as it's one line you can then drop from https://github.com/mapfau/xbmc/commit/b2357611a8c2f81418d61a5911d85db915a8c698